project/core/urls.py

```diff
@@ -22,17 +24,32 @@
 
 from core.router import CiviWikiRouter
 
-urlpatterns = [
-    path("admin/", admin.site.urls),
-    path("", include('accounts.urls.urls')),
-    path("api/", include('accounts.urls.api')),
-    path('api/v1/', include((CiviWikiRouter.urls, 'api'))),
-    path("api/", include(threads)),
-    path("inbox/notifications/", include("notifications.urls", namespace="notifications")),
-    path("favicon.ico", RedirectView.as_view(url="/static/favicon/favicon.ico")),
-    path("favicon-32x32.png", RedirectView.as_view(url="/static/favicon/favicon-32x32.png")),
-    path("apple-touch-icon.png", RedirectView.as_view(url="/static/favicon/apple-touch-icon.png")),
-    path("mstile-144x144.png", RedirectView.as_view(url="/static/favicon/mstile-144x144.png")),
-    path("", include(frontend_views))
-]
-
+urlpatterns = (
```

Comment:
The parenthesis wrapping URLs seems weird to me. Please leave `urlpatterns` as it is:

```python
urlpatterns = [
...
]
```
Concatenate media and static URLs to it:

```python
urlpatterns += <static URL config>
urlpatterns += <media URL config>
```
You can look at this [question](https://stackoverflow.com/questions/66386621/is-it-possible-to-have-a-media-url-and-a-satic-url-in-django) to get some help.

Reply:
OK, I've corrected the comments.